kani/kani.py

```diff
@@ -426,21 +426,21 @@ def load(self, fp: PathLike, **kwargs):
     # ==== internals ====
     def _auto_truncate_message(self, msg: ChatMessage, max_len: int) -> ChatMessage:
         """Mutate the provided message until it is less than *max_len* tokens long."""
-        full_content = msg.content
-        if not full_content:
+        full_text = msg.text
```

Comment:
If you want to be cursed here you can use a walrus 
`if not (full_text := msg.text):`